dodiscover/metrics.py

```diff
@@ -0,0 +1,66 @@
+import networkx as nx
+import numpy as np
+from sklearn.metrics import confusion_matrix
+from sklearn.preprocessing import LabelBinarizer
+
+from ._protocol import Graph
+
+
+def confusion_matrix_networks(
+    true_graph: Graph,
+    pred_graph: Graph,
+):
+    """Compute the confusion matrix comparing a predicted graph from the true graph.
+
+    Converts the graphs into adjacency matrices, which are symmetric.
```

Comment:
A adjacency matrix doesn't need to be symmetric, e.g., when it represents a directed graph.

Reply:
I don't think the confusion matrix is symmetric in general?  Is it not the following?

|TP: True Edges ∩ Detected Edges | FN: True Edges ∩ Undetected Edges|
|FP: False Edges ∩ Detected Edges | TN: False Edges ∩ Undetected Edges|

Reply:
Btw. I was referring to the adjacency matrix here (not confusion matrix).

Reply:
I have updated the docstring.